package.json

```diff
@@ -102,12 +103,12 @@
     "@commitlint/cli": "16.3.0",
     "@commitlint/config-conventional": "16.2.4",
     "@nestjs/schematics": "8.0.11",
-    "@nestjs/testing": "9.3.11",
-    "@nrwl/eslint-plugin-nx": "12.10.1",
+    "@nestjs/testing": "10.2.9",
+    "@nrwl/eslint-plugin-nx": "13.9.4",
     "@nrwl/jest": "12.10.1",
-    "@nrwl/nest": "12.10.1",
+    "@nrwl/nest": "13.9.4",
     "@nrwl/node": "12.10.1",
-    "@nrwl/workspace": "12.10.1",
+    "@nrwl/workspace": "17.0.3",
```

Comment:
I wouldn't recommended updating nx directly in the package.json. Shouldn't we use `nx migrate` (https://nx.dev/core-features/automate-updating-dependencies) as that usually does much more work than just updating dependencies.

Reply:
I'm sorry, I should have read nx's document and shouldn't include nx update in this PR.
I only updated something related to the issue

Reply:
@wvanderdeijl 
Could you re-review this PR?
I need approval from a maintainer to run ci workflow.